mlflow/server/js/src/common/components/Error400View.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import error400Img from '../static/default-error.svg';
+import Colors from '../../experiment-tracking/styles/Colors';
+import Routes from '../../experiment-tracking/routes';
+import { Link } from 'react-router-dom';
+
+export class Error400View extends Component {
```

Comment:
This component looks like almost Error404View. What do you think about creating an ErrorView which takes in an extra prop of errorCode?

Reply:
Sounds good, I'll try it

Reply:
Done.
I created ErrorView and replaced `Error404View` and `Error400View`.
Additionally, `RunNotFoundView` and `PageNotFoundView` have used duplicated code, I unified them.

Reply:
And, since I found that an error message from the server is notified in pop-up balloon, I modified this ErrorView to show just a status code and a status message.

Reply:
![image](https://user-images.githubusercontent.com/6898958/88142698-ecb4f480-cc30-11ea-98f1-5c59bf6f0cee.png)